support/oidc-discovery-provider/config.go

```diff
@@ -224,6 +241,20 @@ func ParseConfig(hclConfig string) (_ *Config, err error) {
 	return c, nil
 }
 
+func unique(items []string) []string {
+	keys := make(map[string]bool)
+	list := make([]string, 0)
```

Comment:
Allocating a zero length slice is a little confusing unless there is a requirement that the function never return a `nil` slice, which isn't typical. I'd rather see:
```
var list []string
```
or
```
list := make([]string, 0, len(items))
```